.gitmodules

```diff
@@ -10,3 +10,7 @@
 [submodule "firmware/third_party/SEGGER_SystemView_Src"]
 	path = firmware/third_party/SEGGER_SystemView_Src
 	url = https://github.com/UBCFormulaElectric/SEGGER_SystemView_Src.git
+[submodule "firmware/third_party/littlefs"]
+	path = firmware/third_party/littlefs
+	url = "git@github.com:littlefs-project/littlefs.git"
+	
```

Comment:
you can unsubmodule littlefs if you are using CPM

Reply:
Do you mean to remove it? I found out that if I remove it, the pull_request test does not pass.

Reply:
![image](https://github.com/UBCFormulaElectric/Consolidated-Firmware/assets/20777515/cc542e1b-7759-4d5f-a8ca-f8a9f917b8da)

Reply:
I see, I had this issue earlier. Perhaps, @gtaharaedmonds can remove it because he is magic submodule remover man.